ios/Wallet/Wallet.swift

```diff
@@ -29,9 +30,17 @@ class Wallet: NSObject {
     func setVerifierPublicKey(publicKeyData: Data) {
         verifierPublicKey = publicKeyData
     }
+    
+    func setTuvaliVersion(_ version: String) {
+        //central?.tuvaliVersion = version
+    }
+    
+    func startScanning(){
+        central?.walletDelegate = self
+    }
 
     func destroyConnection(isSelfDisconnect: Bool){
-        onDeviceDisconnected(isSelfDisconnect: isSelfDisconnect)
+        central?.onDeviceDisconnected(isManualDisconnect: isSelfDisconnect)
```

Comment:
I think we should separate out function that does disconnect Vs function which handles disconnect event.

Here destroyConnection calling onDeviceDisconnected is not making sense.
Can you refer naming conventions from Android side?